### ISSUE TITLE
求助：

### ISSUE THREAD

**Issue by zhangwei5** [mpj1234/ncnn-yolov12-android#4]
您好，我在使用ncnn-yolov12-android/app/src/main /assets里面的模型进行推理时，出现乱框。
![Image](https://github.com/user-attachments/assets/7771b821-e3cc-4fc0-8d35-e7c39086b90c)

代码部分稍作了修改，没有使用安卓部分，ncnn和pnnx都是最新的。
detect函数中printf("out.h = %d, out.w = %d\n", out.h, out.w);这里打印为out.h = 84, out.w = 2100。

另外我在yolov12官方的导出方法中也出现了allback batch axis 233 for operand 40的状况，同时ncnn和pnnx都是最新版。

请问问题大概是出在哪里呢？如果要适配，是需要更改.param文件还是实例yolov12.cpp代码以适应自训练的模型呢？

希望得到您的回答，十分感谢！

**Comment by mpj1234** [comment]
请查看readme，我转换过程中也出现`fallback batch axis 233 for operand ......`，这个并没有影响推理
详细日志：
<details>
```
D:\Anaconda\install\envs\llm\python.exe F:\python\yolov12-main-new\export.py 
FlashAttention is not available on this device. Using scaled_dot_product_attention instead.
Ultralytics 8.3.63 🚀 Python-3.10.14 torch-2.3.1+cu118 CPU (12th Gen Intel Core(TM) i5-12600K)
YOLOv12n summary (fused): 376 layers, 2,542,440 parameters, 0 gradients, 6.0 GFLOPs

PyTorch: starting from 'weights\yolov12n.pt' with input shape (1, 3, 320, 320) BCHW and output shape(s) (1, 84, 2100) (5.3 MB)

TorchScript: starting export with torch 2.3.1+cu118...
TorchScript: export success ✅ 2.0s, saved as 'weights\yolov12n.torchscript' (10.3 MB)

NCNN: starting export with NCNN 1.0.20241226...
NCNN: running 'F:\python\yolov12-main-new\ultralytics\pnnx.exe weights\yolov12n.torchscript ncnnparam=weights\yolov12n_ncnn_model\model.ncnn.param ncnnbin=weights\yolov12n_ncnn_model\model.ncnn.bin ncnnpy=weights\yolov12n_ncnn_model\model_ncnn.py pnnxparam=weights\yolov12n_ncnn_model\model.pnnx.param pnnxbin=weights\yolov12n_ncnn_model\model.pnnx.bin pnnxpy=weights\yolov12n_ncnn_model\model_pnnx.py pnnxonnx=weights\yolov12n_ncnn_model\model.pnnx.onnx fp16=0 device=cpu inputshape="[1, 3, 320, 320]"'
pnnxparam = weights\yolov12n_ncnn_model\model.pnnx.param
pnnxbin = weights\yolov12n_ncnn_model\model.pnnx.bin
pnnxpy = weights\yolov12n_ncnn_model\model_pnnx.py
pnnxonnx = weights\yolov12n_ncnn_model\model.pnnx.onnx
ncnnparam = weights\yolov12n_ncnn_model\model.ncnn.param
ncnnbin = weights\yolov12n_ncnn_model\model.ncnn.bin
ncnnpy = weights\yolov12n_ncnn_model\model_ncnn.py
fp16 = 0
optlevel = 2
device = cpu
inputshape = [1,3,320,320]f32
inputshape2 = 
customop = 
moduleop = 
get inputshape from traced inputs
inputshape = [1,3,320,320]f32
############# pass_level0
inline module = torch.nn.modules.linear.Identity
inline module = ultralytics.nn.modules.block.A2C2f
inline module = ultralytics.nn.modules.block.AAttn
inline module = ultralytics.nn.modules.block.ABlock
inline module = ultralytics.nn.modules.block.Bottleneck
inline module = ultralytics.nn.modules.block.C3k
inline module = ultralytics.nn.modules.block.C3k2
inline module = ultralytics.nn.modules.block.DFL
inline module = ultralytics.nn.modules.conv.Concat
inline module = ultralytics.nn.modules.conv.Conv
inline module = ultralytics.nn.modules.conv.DWConv
inline module = ultralytics.nn.modules.head.Detect
inline module = torch.nn.modules.linear.Identity
inline module = ultralytics.nn.modules.block.A2C2f
inline module = ultralytics.nn.modules.block.AAttn
inline module = ultralytics.nn.modules.block.ABlock
inline module = ultralytics.nn.modules.block.Bottleneck
inline module = ultralytics.nn.modules.block.C3k
inline module = ultralytics.nn.modules.block.C3k2
inline module = ultralytics.nn.modules.block.DFL
inline module = ultralytics.nn.modules.conv.Concat
inline module = ultralytics.nn.modules.conv.Conv
inline module = ultralytics.nn.modules.conv.DWConv
inline module = ultralytics.nn.modules.head.Detect

----------------

############# pass_level1
############# pass_level2
############# pass_level3
############# pass_level4
############# pass_level5
############# pass_ncnn
fallback batch axis 233 for operand 42
fallback batch axis 233 for operand 43
fallback batch axis 233 for operand 44
fallback batch axis 233 for operand 45
fallback batch axis 233 for operand 46
fallback batch axis 233 for operand 47
fallback batch axis 233 for operand 48
fallback batch axis 233 for operand 49
fallback batch axis 233 for operand 50
fallback batch axis 233 for operand 51
fallback batch axis 233 for operand 52
fallback batch axis 233 for operand 53
fallback batch axis 233 for operand 55
fallback batch axis 233 for operand 57
fallback batch axis 233 for operand 59
fallback batch axis 233 for operand 60
fallback batch axis 233 for operand 61
fallback batch axis 233 for operand 78
fallback batch axis 233 for operand 79
fallback batch axis 233 for operand 80
fallback batch axis 233 for operand 81
fallback batch axis 233 for operand 82
fallback batch axis 233 for operand 83
fallback batch axis 233 for operand 84
fallback batch axis 233 for operand 85
fallback batch axis 233 for operand 86
fallback batch axis 233 for operand 87
fallback batch axis 233 for operand 88
fallback batch axis 233 for operand 89
fallback batch axis 233 for operand 91
fallback batch axis 233 for operand 93
fallback batch axis 233 for operand 95
fallback batch axis 233 for operand 96
fallback batch axis 233 for operand 97
fallback batch axis 233 for operand 114
fallback batch axis 233 for operand 115
fallback batch axis 233 for operand 116
fallback batch axis 233 for operand 117
fallback batch axis 233 for operand 118
fallback batch axis 233 for operand 119
fallback batch axis 233 for operand 120
fallback batch axis 233 for operand 121
fallback batch axis 233 for operand 122
fallback batch axis 233 for operand 123
fallback batch axis 233 for operand 124
fallback batch axis 233 for operand 125
fallback batch axis 233 for operand 127
fallback batch axis 233 for operand 129
fallback batch axis 233 for operand 131
fallback batch axis 233 for operand 132
fallback batch axis 233 for operand 133
fallback batch axis 233 for operand 150
fallback batch axis 233 for operand 151
fallback batch axis 233 for operand 152
fallback batch axis 233 for operand 153
fallback batch axis 233 for operand 154
fallback batch axis 233 for operand 155
fallback batch axis 233 for operand 156
fallback batch axis 233 for operand 157
fallback batch axis 233 for operand 158
fallback batch axis 233 for operand 159
fallback batch axis 233 for operand 160
fallback batch axis 233 for operand 161
fallback batch axis 233 for operand 163
fallback batch axis 233 for operand 165
fallback batch axis 233 for operand 167
fallback batch axis 233 for operand 168
fallback batch axis 233 for operand 169
fallback batch axis 233 for operand pnnx_expr_618_mul(53,0.176777)
fallback batch axis 233 for operand pnnx_expr_614_sub(pnnx_expr_618_mul(53,0.176777),55)
fallback batch axis 233 for operand pnnx_expr_614_exp(sub(pnnx_expr_618_mul(53,0.176777),55))
fallback batch axis 233 for operand pnnx_expr_612_div(pnnx_expr_614_exp(sub(pnnx_expr_618_mul(53,0.176777),55)),57)
fallback batch axis 233 for operand pnnx_expr_521_mul(89,0.176777)
fallback batch axis 233 for operand pnnx_expr_517_sub(pnnx_expr_521_mul(89,0.176777),91)
fallback batch axis 233 for operand pnnx_expr_517_exp(sub(pnnx_expr_521_mul(89,0.176777),91))
fallback batch axis 233 for operand pnnx_expr_515_div(pnnx_expr_517_exp(sub(pnnx_expr_521_mul(89,0.176777),91)),93)
fallback batch axis 233 for operand pnnx_expr_424_mul(125,0.176777)
fallback batch axis 233 for operand pnnx_expr_420_sub(pnnx_expr_424_mul(125,0.176777),127)
fallback batch axis 233 for operand pnnx_expr_420_exp(sub(pnnx_expr_424_mul(125,0.176777),127))
fallback batch axis 233 for operand pnnx_expr_418_div(pnnx_expr_420_exp(sub(pnnx_expr_424_mul(125,0.176777),127)),129)
fallback batch axis 233 for operand pnnx_expr_327_mul(161,0.176777)
fallback batch axis 233 for operand pnnx_expr_323_sub(pnnx_expr_327_mul(161,0.176777),163)
fallback batch axis 233 for operand pnnx_expr_323_exp(sub(pnnx_expr_327_mul(161,0.176777),163))
fallback batch axis 233 for operand pnnx_expr_321_div(pnnx_expr_323_exp(sub(pnnx_expr_327_mul(161,0.176777),163)),165)
NCNN: export success ✅ 1.4s, saved as 'weights\yolov12n_ncnn_model' (9.8 MB)

Export complete (3.8s)
Results saved to F:\python\yolov12-main-new\weights
Predict:         yolo predict task=detect model=weights\yolov12n_ncnn_model imgsz=320  
Validate:        yolo val task=detect model=weights\yolov12n_ncnn_model imgsz=320 data=None  
Visualize:       https://netron.app

进程已结束，退出代码为 0

```
</details>
